scapy/layers/usb.py

```diff
@@ -8,20 +9,22 @@
 Default USB frames & Basic implementation
 """
 
-# TODO: support USB headers for Linux and Darwin (usbmon/netmon)
+# TODO: support USB headers for Darwin (netmon)
 # https://github.com/wireshark/wireshark/blob/master/epan/dissectors/packet-usb.c  # noqa: E501
 
 import re
 import subprocess
+from enum import Enum, IntEnum
 
 from scapy.config import conf
 from scapy.consts import WINDOWS
 from scapy.compat import chb, plain_str
-from scapy.data import MTU, DLT_USBPCAP
+from scapy.data import MTU, DLT_USBPCAP, DLT_USB_LINUX, DLT_USB_LINUX_MMAPPED
 from scapy.error import warning
 from scapy.fields import ByteField, XByteField, ByteEnumField, LEShortField, \
     LEShortEnumField, LEIntField, LEIntEnumField, XLELongField, \
-    LenField
+    LenField, PacketLenField, LELongField, EnumField, ConditionalField, \
+    MultipleTypeField, StrLenField, cast, I
```

Comment:
You shouldn't need `cast` or `I` in this case, but you should never need to import `I`